src/Symfony/README.md

```diff
@@ -0,0 +1,43 @@
+# Async AWS Symfony Bundle
+
+![](https://github.com/async-aws/symfony-bundle/workflows/Tests/badge.svg?branch=master)
+![](https://github.com/async-aws/symfony-bundle/workflows/BC%20Check/badge.svg?branch=master)
+
+A small SymfonyBundle that helps with configuration and autowiring.
+
+## Install
+
+```cli
+composer require async-aws/symfony-bundle
+```
+
+## Configure
+
+```yaml
+async_aws:
+    config: # Will be merged with other configuration
+        region: eu-central-1
+
+    services:
+        ses: ~ # This will complain if we dont have that package installed
+        sqs:
+            config:
+                region: us-west-1
+        foobar:
+            type: sqs
```

Comment:
Are we happy with this config and autowiring?

Reply:
Nice !